fuzz/fuzz_targets/1_message_from_bytes.rs

```diff
@@ -14,21 +15,25 @@ fuzz_target!(|data: &[u8]| {
         Ok(message) => {
             // not so interesting because it mostly tests external compression code?
             // let _ = message.compress(CompressionAlgorithm::ZIP);
-            let _ = message.clone().decompress();
+            let _ = Message::from_bytes(data).unwrap().decompress();
 
             // FUZZER RESULT this can panic on some inputs
             // finding RPG-19 in ROS report 2024, fixed with 0.14.1
-            let _ = message.decrypt_with_password(|| "bogus_password".into());
+            if let Ok(mut dec) = message.decrypt_with_password(&Password::from("bogus_password")) {
+                let _ = dec.as_data_vec();
+            }
+
+            let _ = Message::from_bytes(data).unwrap().is_one_pass_signed();
+            let _ = Message::from_bytes(data).unwrap().is_literal();
+            let _ = Message::from_bytes(data).unwrap().as_data_string();
 
-            let _ = message.clone().is_one_pass_signed();
-            let _ = message.clone().is_literal();
-            let _ = message.clone().get_literal();
             // attempts decompression for some message types
-            let _ = message.clone().get_content();
+            // FIXME: manually try to unwrap the message? (esp. decompress?)
```

Comment:
maybe sth like 
- if compressed try decompress
- if encrypted try decrypt
- finally just read the content